lib/ad_localize/runner.rb

```diff
@@ -7,29 +7,41 @@ def initialize
     end
 
     def run(args = ARGV)
-      LOGGER.log(:info, :green, "OPTIONS : #{options}")
-      input_files = (args + [options.dig(:drive_key)]).compact # drive_key can be nil
-      if input_files.length.zero?
-        LOGGER.log(:error, :red, "No CSV to parse. Use option -h to see how to use this script")
-      else
-        file_to_parse = args.first
-        LOGGER.log(:warn, :yellow, "Only one CSV can be treated - the priority goes to #{file_to_parse}") if input_files.length > 1
-        if args.empty?
-          options[:drive_file] = CsvFileManager.download_from_drive(options.dig(:drive_key), options.dig(:sheet_id), options.dig(:use_service_account))
-          file_to_parse = options.dig(:drive_file)
+        LOGGER.log(:info, :green, "OPTIONS : #{options}")
+        input_files = args
+        has_drive_key = options.dig(:drive_key)
+        raise 'No CSV to parse. Use option -h to see how to use this script' if input_files.length.zero? and !has_drive_key
+
+        files_to_parse = Array.new
+        if has_drive_key
+            LOGGER.log(:warn, :yellow, 'CSV file are ignored with the drive key option') if input_files.length > 1
+            options[:drive_file] = CsvFileManager.download_from_drive(options.dig(:drive_key), options.dig(:sheet_id), options.dig(:use_service_account))
+            files_to_parse.push(options.dig(:drive_file))
+        else
+            files_to_parse += input_files
         end
-        if CsvFileManager.csv?(file_to_parse)
-          export(file_to_parse)
+        LOGGER.log(:debug, :black, "FILES: #{files_to_parse}")
+        if files_to_parse.length > 1
+            export_all(files_to_parse)
         else
-          LOGGER.log(:error, :red, "#{file_to_parse} is not a csv. Make sure to enable \"Allow external access\" in sharing options or use a service account.")
+            export(files_to_parse.first)
         end
-        CsvFileManager.delete_drive_file(options[:drive_file]) if options[:drive_file]
-      end
     end
 
     private
 
-    def export(file)
+    def export_all(files)
+        files.each do |file|
+            if CsvFileManager.csv?(file)
```

Comment:
There is a [method](https://github.com/applidium/ad_localize/blob/144f1cb697b94d386027a667df5ce58a0cfc99b8/lib/ad_localize/csv_file_manager.rb#L36) in CsvFileManager to select the CSV files in an array. Can you use it to first select the right files then export each of them. 

Also, can you update the wording in `select_csvs(files)` to include the wording "Make sure to enable \"Allow external access\" in sharing options or use a service account.". The log [here](https://github.com/applidium/ad_localize/pull/39/commits/c607ecebdee7eecfac7aef9b13e5ad288cbbd846#diff-0e657f985321c2ae69b2d93cffa632cfR38) won't be necessary anymore.